app/pages/nonsipp/memberdetails/NewMember.scala

```diff
@@ -19,28 +19,16 @@ package pages.nonsipp.memberdetails
 import utils.RefinedUtils.RefinedIntOps
 import models.SchemeId.Srn
 import play.api.libs.json.JsPath
-import models.UserAnswers
-import viewmodels.models.MemberState
+import viewmodels.models.Flag
 import config.Refined.Max300
 import pages.QuestionPage
 
-// todo remove in favour of soft deletion
-case class MemberStatus(srn: Srn, index: Max300) extends QuestionPage[MemberState] {
```

Comment:
MemberStatus was deprecated but all the call sites were where I wanted to add NewMember so changed the name